src/test/java/teammates/test/MockLogsProcessor.java

```diff
@@ -105,4 +131,40 @@ public void createFeedbackSessionLog(String courseId, String email, String fsNam
         return feedbackSessionLogs;
     }
 
+    /**
+     * Allows mocking of {@link GeneralLogEntry}.
+     */
+    public static class MockGeneralLogEntry extends GeneralLogEntry {
+        private final String actionClass;
+        private final UserInfoParams userInfoParams;
+        private final String logEvent;
+        private final String exceptionClass;
```

Comment:
Are any of these used in any test?

Reply:
Those were used in the tests in the last few commits. It is not used now. They are removed. `MockGeneralLogEntry` is also removed for now as it is now used in tests as well.